README.md

```diff
@@ -151,9 +151,9 @@ not just visualize metrics.
   
  - New query engine, supporting statistical functions.  
  - Fixed security issues identified by Red4Sec.com and Synacktiv.  
- - New Data Collection Modules: `rethinkdbs`, `proxysql`, `litespeed`, `uwsgi`, `unbound`, `powerdns`, `dockerd`, `puppet`, `logind`, `adaptec_raid`, `megacli`, `spigotmc`, `boinc`, `w1sensor`, `monit`, `linux_power_supplies`.  
- - Improved Data Collection Modules: `statsd.plugin`, `apps.plugin`, `freeipmi.plugin`, `proc.plugin`, `diskspace.plugin`, `freebsd.plugin`, `python.d.plugin`, `web_log`, `nginx_plus`, `ipfs`, `fail2ban`, `ceph`, `elasticsearch`, `nginx_plus`,  `redis`,   
- `beanstalk`, `mysql`, `varnish`, `couchdb`, `phpfpm`, `apache`, `icecast`, `mongodb`, `postgress`, `elasticsearch`, `mdstat`, `openvpn_log`, `snmp`, `nut`.  
+ - New Data Collection Modules: `[rethinkdbs](collectors/python.d.plugin/rethinkdbs/)`, `[proxysql](collectors/python.d.plugin/proxysql/)`, `[litespeed](collectors/python.d.plugin/litespeed/)`, `[uwsgi](collectors/python.d.plugin/uwsgi/)`, `[unbound](collectors/python.d.plugin/unbound/)`, `[powerdns](collectors/python.d.plugin/powerdns/)`, `[dockerd](collectors/python.d.plugin/dockerd/)`, `[puppet](collectors/python.d.plugin/puppet/)`, `[logind](collectors/python.d.plugin/logind/)`, `[adaptec_raid](collectors/python.d.plugin/adaptec_raid/)`, `[megacli](collectors/python.d.plugin/megacli/)`, `[spigotmc](collectors/python.d.plugin/spigotmc/)`, `[boinc](collectors/python.d.plugin/boinc/)`, `[w1sensor](collectors/python.d.plugin/w1sensor/)`, `[monit](collectors/python.d.plugin/monit/)`, `[linux_power_supplies](collectors/python.d.plugin/linux_power_supply/)`.  
```

Comment:
If you put stuff in backticks (`) then markdown treats it as a code part and won't render it as a link. Also links put inside backticks aren't very friendly from UX perspective.